.changeset/few-mayflies-change.md

```diff
@@ -0,0 +1,11 @@
+---
+"@balancer/sdk": minor
+---
+
+Replace dataQueries with multicallV3:
+
+- Add multicallV3 ABI
+- Add manual batching (instead of Viems) so we can maintain custom blockNo
```

Comment:
If I understood correctly, you're saying you're using multicallV3 directly because viem's multicall does not allow you to set a blockNumber - is that right?
Looking at the previous implementation, it seems that we're able to set the blockNumber, so I'm assuming I misunderstood something 🤔 
So, my question is: why aren't we using viem's multicall?

Reply:
We do [allow you to set a block number](https://github.com/wagmi-dev/viem/blob/main/src/actions/public/multicall.ts#L112) FWIW. It's not documented, but we should document it 😅